packages/components-react/src/heading/Heading.tsx

```diff
@@ -0,0 +1,47 @@
+import { Heading as UtrechtHeading } from '@utrecht/component-library-react/dist/css-module';
+import { HTMLAttributes, PropsWithChildren, ReactNode } from 'react';
+
+type HeadingLevels = 1 | 2 | 3 | 4 | 5 | 6;
+
+export interface LuxHeadingProps extends HTMLAttributes<HTMLHeadingElement> {
+  appearance?: HeadingLevels;
+  level: HeadingLevels;
+}
+
+export const LuxHeading = ({
+  children,
+  level = 6,
+  appearance = level,
+  ...props
+}: PropsWithChildren<LuxHeadingProps>): ReactNode => {
+  const _level = level > 6 || level < 1 ? 6 : level;
+  const _appearance = appearance > 6 || appearance < 1 ? 6 : appearance;
+  return (
+    <UtrechtHeading
+      level={_level}
+      {...(_appearance ? { appearance: `utrecht-heading-${_appearance}` } : {})}
+      {...props}
+    >
+      {children}
+    </UtrechtHeading>
+  );
+};
+
+export const LuxHeading1 = ({ ...props }: PropsWithChildren<HTMLAttributes<HTMLHeadingElement>>): ReactNode => (
```

Comment:
The `appearance` props are not necessary here, as they are implicitly equal to the `level` prop. Not sure if leaving it or adding it is the best way to go. 

I'm genuinely 50-50 about it.